pyproject.toml

```
@@ -1,15 +1,15 @@
 
 [build-system]
-requires = ["setuptools>=64", "setuptools-scm>=8", "packaging"]
+requires = ["setuptools>=64", "setuptools-scm>=8", "packaging", "saxonche"]
 build-backend = "setuptools.build_meta"
 
 [project]
 name = "data_dictionary"
 description ="The Data Dictionary is the implementation of the Data Model of ITER's Integrated Modelling & Analysis Suite (IMAS)"
 authors = [{name = "ITER Organization", email = "imas-support@iter.org"}]
 readme = {file = "README.md", content-type = "text/markdown"}
-license = {file = "LICENSE.md"}
-keywords = ["Data Dictionary", "IDS"]
+license = {file = "LICENSE-LGPL"}
```

Not sure how this will be used or where it will appear but the DD and documentation aren't LGPL so probably better to go with:
```suggestion
license = {file = "LICENSE-CC-BY-ND"}
```